### PR TITLE
homepage

### PR DESCRIPTION
**Pull request by prath-dev**
Home page Added .
Live Preview: https://deploy-preview-2--mystores.netlify.app/views/

### CODE REVIEW

File: Components/Css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    margin-top: 1rem;
+}
+
+.home-images img {
+    height: 22rem;
+}
+
+.categories {
+    display: flex;
+    justify-content: space-evenly;
```
Comment (vikrantkode) [inline Components/Css/homepage.css:13]: -you can made two containers and then give spaces......like logo at left....other header icons on left

File: Components/Css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.shopnow {
+    display: flex;
+    justify-content: center;
+}
+
+.button-solid-secondary {
+    background-color: white;
+}
```
Comment (vikrantkode) [inline Components/Css/homepage.css:29]: give variable naming to colors and then use it

File: Components/Css/cart.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
+.cart-container {
+    display: flex;
+    justify-content: center;
+    margin-top: 3rem;
+}
+
+.horizontal-card {
+    margin-right: 3rem;
+    height: 13rem;
+    box-shadow: 5px 10px 18px #888888;
```
Comment (shubhmehra) [inline Components/Css/cart.css:10]: please use colour from var for consistency!

File: Components/Css/cart.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    color: black;
+    border: 2px solid blue;
+}
+
+.quantity-value {
+    margin-left: 2rem;
+}
+
+.price-card {
+    box-shadow: 5px 10px 18px #888888;
```
Comment (shubhmehra) [inline Components/Css/cart.css:45]: same colour changes

**Review by vikrantkode — COMMENTED** [review]
- Some change suggested.....
- rest is good to me

File: Views/cart.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+              </a>
+        <a href="..//Views/wishlist.html">WishList<span class="material-icons-outlined">
+       favorite_border
+        </span>
+          </a>
+        <a href="..//Views/login.html">Login</a>
+    </nav>
+    <div class="cart-container">
+        <div class="horizontal-card">
+            <img src="../Components/Images/tshirt.jpg" alt="">
```
Comment (shubhmehra) [inline Views/cart.html:30]: please always try to add proper alt messages for good accessibility of your side

File: Views/cart.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+                <span>Total Amount</span>
+                <span>₹750</span>
+            </div>
+            <div class="price-details">
+                <span>You will save ₹200 on this order</span>
+            </div>
+            <button class="place-order-btn">Place Order</button>
+        </div>
+    </div>
+
```
Comment (shubhmehra) [inline Views/cart.html:71]: you can use prettier for format your code...
it will remove this unwanted lines

File: Views/index.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+        <a href="..//Views/wishlist.html">WishList<span class="material-icons-outlined">
+       favorite_border
+        </span>
+          </a>
+        <a href="..//Views/login.html">Login</a>
+    </nav>
+    <div class="home-container">
+        <div class="home-images">
+            <a href="../Views/products.html">
+                <img src="..//Components/Images/homepage.gif" alt="">
```
Comment (shubhmehra) [inline Views/index.html:31]: same here...
Please update alt messages

**Review by shubhmehra — CHANGES_REQUESTED** [review]
Code format structure & alt messages need to be fixed..
otherwise code looks good

**Review by Balakumar3018 — APPROVED** [review]
Everything looks good
suggestion - make it responsive for home image

**Comment by prath-dev** [comment]
@vikrantkode @shubhmehra  thansks for your suggestion i will do changes in second version of ecommerce.

**Pull request merged**